docs/DeveloperGuide.md

```diff
@@ -283,9 +284,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. GradPad shows an error message.
+
+      Use case resumes at step 2.
+      
+**Use case : Add a Module**
+
+**MSS**
+
+1. User requests to list Modules
+2. GradPad shows a list of Modules
+3. User requests to add a specific Module into the list
+4. GradPad adds the module
+
+    Use case ends.
+    
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
```

Comment:
Maybe an empty list shouldn't affect the addition of a new module? If not a user can never add his first mod since the list will be empty then LOL